### PR TITLE
Document kuryr_openstack_ca option

### PR DESCRIPTION
**Pull request by dulek**
This option is only documented on readme in openshift-ansible, let's
make sure it is described here as well.

### CODE REVIEW

**Comment by dulek** [comment]
Okay, I guess I cannot mention the BZ.

File: install_config/topics/ocp_osp_provisioning.adoc [hunk 1]
```diff
@@ -162,6 +162,9 @@ kuryr_openstack_public_net_id: *<public_ID>*
 openshift_kuryr_subnet_driver: namespace
 openshift_kuryr_sg_driver: namespace
 
+# If OpenStack cloud endpoints are accessible over HTTPS set option below.
```
Comment (kalexand-rh) [inline install_config/topics/ocp_osp_provisioning.adoc]:
```suggestion
# If OpenStack cloud endpoints are accessible over HTTPS, provide the
CA certificate
```

Comment (dulek) [reply install_config/topics/ocp_osp_provisioning.adoc]: Fixed, thanks!

**Comment by luis5tb** [comment]
Please add the explanation in the table below where the different parameters are explained

**Comment by dulek** [comment]
> Please add the explanation in the table below where the different parameters are explained

Done, thanks for pointing it out!

**Comment by maxwelldb** [comment]
Hi @luis5tb @dulek -- this PR needs a QE :+1: for us to merge it. Do you have anyone in mind who could do that?

**Comment by eurijon** [comment]
> Hi @luis5tb @dulek -- this PR needs a QE +1 for us to merge it. Do you have anyone in mind who could do that?

Hey, this looks good to me!

**Pull request merged**